docs/content/Configuration/Databases/AWS-Athena.md

```diff
@@ -32,6 +33,7 @@ CUBEJS_AWS_S3_OUTPUT_LOCATION=s3://my-athena-output-bucket
 | `CUBEJS_AWS_SECRET`             | The AWS Secret Access Key to use for database connections         | A valid AWS Secret Access Key          |    ✅    |
 | `CUBEJS_AWS_REGION`             | The AWS region of the Cube.js deployment                          | [A valid AWS region][aws-docs-regions] |    ✅    |
 | `CUBEJS_AWS_S3_OUTPUT_LOCATION` | The S3 path to store query results made by the Cube.js deployment | A valid S3 path                        |    ❌    |
+| `CUBEJS_AWS_ATHENA_WORKGROUP`   | The name of the workgroup in which the query is being started     | A valid Athena Workgroup               |    ❌    |
```

Comment:
Might be good to link "A valid Athena Workgroup" out to https://docs.aws.amazon.com/athena/latest/ug/workgroups-benefits.html